cfn-template/deeplearning.template

```diff
@@ -13,19 +72,19 @@
       "Default" : "1"
     },
     "InstanceType" : {
-      "Description" : "The EC2 instance type for workers.For GPUs choose g3.xx, p2.xx or p3.xx.",
+      "Description" : "The EC2 instance type for workers. For latest GPUs choose p3.xx",
       "Type" : "String",
-      "Default" : "p3.2xlarge",
+      "Default" : "p3.16xlarge",
       "AllowedValues" : [
         "p2.16xlarge",
         "p2.8xlarge",
         "p2.xlarge",
         "p3.2xlarge",
         "p3.8xlarge",
         "p3.16xlarge",
-        "g3.16xlarge",
-        "g3.8xlarge",
-        "g3.4xlarge",
+        "p3dn.24xlarge",
```

Comment:
g2 are no longer supported by DLAMI so they should be removed.
```
"p3dn.24xlarge",
"g3.16xlarge",
"g3.8xlarge",
"g3.4xlarge",
```
Please using these values here.